stock_picking_partner_note/models/stock_picking.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    note = fields.Html(compute="_compute_note", store=True)
+
+    @api.depends("partner_id")
```

Comment:
```suggestion
    @api.depends("picking_type_id.partner_note_type_ids", "partner_id.stock_picking_note_ids.name/description")
```

Reply:
We don't want to update note when `picking_type_id.partner_note_type_ids` change

Reply:
Then why do we rely on it in the compute method?
Also, I would depend on the state, as when picking is "done", we do not want to update the note anymore.

Reply:
The content of the note by default is taken from partner and recomputed if and only if partner change.